sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -226,7 +234,7 @@ make_kernel_bundle<backend::ext_oneapi_level_zero,
     const context &Context)
 ```
 </td>
-<td>Constructs a SYCL kernel_bundle instance from a Level-Zero <code>ze_module_handle_t</code>. The <code>Context</code> argument must be a valid SYCL context encapsulating a Level-Zero context. The Level-Zero module must be fully linked (i.e. not require further linking through <a href="https://spec.oneapi.com/level-zero/latest/core/api.html?highlight=zemoduledynamiclink#_CPPv419zeModuleDynamicLink8uint32_tP18ze_module_handle_tP28ze_module_build_log_handle_t"><code>zeModuleDynamicLink</code></a>), and thus the SYCL kernel_bundle is created in the "executable" state.</td>
+<td>Constructs a SYCL kernel_bundle instance from a Level-Zero <code>ze_module_handle_t</code>. The <code>Context</code> argument must be a valid SYCL context encapsulating a Level-Zero context. The Level-Zero module should be created on the same context. The Level-Zero module must be fully linked (i.e. not require further linking through <a href="https://spec.oneapi.com/level-zero/latest/core/api.html?highlight=zemoduledynamiclink#_CPPv419zeModuleDynamicLink8uint32_tP18ze_module_handle_tP28ze_module_build_log_handle_t"><code>zeModuleDynamicLink</code></a>), and thus the SYCL kernel_bundle is created in the "executable" state. The <code>Ownership</code> input structure member specifies if the SYCL runtime should take ownership of the passed native handle. The default behavior is to transfer the ownership to the SYCL runtime. See section 4.4 for details. If the behavior is "transfer" then runtime is going to destroy the input Level-Zero module, and hence the application must not to have any outstanding <code>ze_kernel_handle_t</code> handles to the underlying <code>ze_module_handle_t</code> by the time this interoperability <code>kernel_bundle</code> destructor is called.</td>

```

Comment:
```suggestion
<td>Constructs a SYCL kernel_bundle instance from a Level-Zero <code>ze_module_handle_t</code>. The <code>Context</code> argument must be a valid SYCL context encapsulating a Level-Zero context, and the Level-Zero module must be created on the same context. The Level-Zero module must be fully linked (i.e. not require further linking through <a href="https://spec.oneapi.com/level-zero/latest/core/api.html?highlight=zemoduledynamiclink#_CPPv419zeModuleDynamicLink8uint32_tP18ze_module_handle_tP28ze_module_build_log_handle_t"><code>zeModuleDynamicLink</code></a>), and thus the SYCL kernel_bundle is created in the "executable" state. The <code>Ownership</code> input structure member specifies if the SYCL runtime should take ownership of the passed native handle. See section 4.4 for details. If the behavior is "transfer" then the runtime is going to destroy the input Level-Zero module, and hence the application must not have any outstanding <code>ze_kernel_handle_t</code> handles to the underlying <code>ze_module_handle_t</code> by the time this interoperability <code>kernel_bundle</code> destructor is called.</td>
```

* I changed "should be created on the same context" to "must be created on the same context".

* I removed "The default behavior is to transfer the ownership to the SYCL runtime" because there is no default behavior here.  The `Ownership` member does not have a default value.  Please update the descriptions of the other `make_` functions to remove this wording also.  I think I made this comment also in #4560, but it somehow fell through the cracks.  (Sorry, I approved that PR without checking on this.)

* I made a few other changes to fix grammar, etc.

BTW, I think it's generally bad style to have such long lines.  Why not break this up into several lines, where each line is 80 columns or less?  Really long lines like this make it harder to review.  I think we restrict line length to 80 columns for code, and that  should apply to specs also.

Reply:
I followed your suggestion except the "default value", which is there.